package.json

```diff
@@ -1,73 +1,65 @@
 {
-  "name": "Arronax",
+  "name": "arronax",
   "version": "0.0.2",
   "description": "Blockchain analytics and business intelligence tool for the Tezos chain.",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack-dev-server --mode development --inline --hot --open",
-    "serve": "webpack-dev-server --mode development --inline --hot",
```

Comment:
Please create a script comparable to this. Something that builds and runs, but does not force open a browser